kitsune/messages/jinja2/messages/inbox.html

```diff
@@ -1,53 +1,61 @@
 {% extends "messages/base.html" %}
 {% from "messages/includes/macros.html" import avatar_link, name_link %}
 {% set title = _('Inbox') %}
-{% set crumbs = [(None, title)] %}
+{% set crumbs = [(url('messages.inbox'), _('Messages')),
+                 (None, title)] %}
 {% set active_tab = 'inbox' %}
 
 {% block content %}
-  <div class="sumo-page-section">
+<div class="sumo-page-section">
     <article id="inbox" class="main">
       <h1 class="sumo-page-heading">{{ title }}</h1>
       <div class="actions sumo-button-wrap">
         <a class="sumo-button primary-button" href="{{ url('messages.new') }}">{{ _('New Message') }}</a>
       </div>
-
-      {% if msgs %}
-        <form action="{{ url('messages.bulk_action') }}" method="POST">
-          <div class="sumo-button-wrap">
-            <input class="sumo-button" type="submit" name="mark_read" value="{{ _('Mark Selected as Read') }}">
-            <input class="sumo-button" type="submit" name="delete" value="{{ _('Delete Selected') }}">
-          </div>
-
-          <ol class="message-list">
-            {% for message in msgs.object_list %}
-              <li
-                class="message-list--item
-                  {% if message.unread %}unread{% endif %}
-                  {% if message.replied %}replied{% endif %}"
-              >
-                <div class="field checkbox no-label">
-                  <input type="checkbox" name="id" value="{{ message.id }}" id="id_checkbox_{{ message.id }}">
-                  <label for="id_checkbox_{{ message.id }}"></label>
+    {% if msgs %}
+    <form action="{{ url('messages.bulk_action') }}" method="POST">
+      <div class="sumo-button-wrap">
+        <input class="sumo-button" type="submit" name="mark_read" value="{{ _('Mark Selected as Read') }}">
+        <input class="sumo-button" type="submit" name="delete" value="{{ _('Delete Selected') }}">
```

Comment:
Do we still need the `Delete Selected` button? I'm thinking that with the ability to delete individual inbox messages, and also since it'd be less clicks to use the individual delete buttons rather than selecting each row and then pushing the `Delete Selected` button, we can get rid of it?

Now we have the ability to delete individual inbox messages with the new delete button for each row, and also

Reply:
Agreed.